tests/stub/routing/scripts/v3/writer_tx_with_unexpected_interruption_on_status_check.script

```diff
@@ -6,18 +6,18 @@ A: HELLO {"{}": "*"}
 {{
     C: BEGIN {}
     S: SUCCESS {}
-    C: RUN "RETURN 1 as n" {} {}
+    C: RUN "RETURN 1 AS n" {} {}
     S: SUCCESS {"fields": ["n"]}
     C: PULL_ALL
     S: SUCCESS {"type": "w"}
     C: COMMIT
     S: SUCCESS {}
-    C: RESET
-    S: SUCCESS {}
+    IF: #RESET_ON_POOL_RETURN#
+        A: RESET
```

Comment:
I didn't know we have if in the stub-script. Nice.

Reply:
:rocket: Introduced, together with Python blocks, in https://github.com/neo4j-drivers/testkit/pull/542.
I can recommend giving the [stub server's README](https://github.com/neo4j-drivers/testkit/blob/5.0/boltstub/README.md) a read for the most up-to date features of stub script.